orquesta/utils/parameters.py

```diff
@@ -61,12 +64,19 @@ def parse_inline_params(s, preserve_order=True):
         v = re.sub("^'", '', v)
         v = re.sub("'$", '', v)
 
-        if v.lower() == 'true' or v.lower() == 'false':
-            v = v.lower()
+        quotes_in_string = False
+        if v != "":
+            # Check if param is a JSON string becuse it can be handled by json.loads()
+            curly_brace_in_param = bool(v[0] == '{' and v[-1] == '}')
+            quotes_in_string = bool(quotes_in_param and not curly_brace_in_param)
+        is_bool_value = bool(v.lower() == 'true' or v.lower() == 'false')
```

Comment:
Also, you don't need to explicitly put the expression in `bool`.  It's already evaluated to boolean.  You can just do 

quotes_in_param = re.findall(REGEX_VALUE_IN_QUOTES, v) or re.findall(REGEX_VALUE_IN_APOSTROPHES, v)
curly_brace_in_param = (v[0] == '{' and v[-1] == '}')
quotes_in_string = quotes_in_param and not curly_brace_in_param
is_bool_value = (v.lower() == 'true' or v.lower() == 'false')